jwst/jump/twopoint_difference.py

```diff
@@ -187,28 +186,27 @@ def get_clipped_median(num_differences, diffs_to_ignore, differences, sorted_ind
     # ignore largest value and number of CRs found when finding new median
     # Check to see if this is a 2-D array or 1-D
     if sorted_index.ndim > 1:
-
         # Get the index of the median value always excluding the highest value
-        pixel_med_index = sorted_index[:, :, int((num_differences - 1) / 2)]
-        # Get the row and column of each pixel.
-        row, col = np.indices(pixel_med_index.shape)
+        row, col = np.indices(diffs_to_ignore.shape)
+        pixel_med_index = sorted_index[row, col, (num_differences - (diffs_to_ignore[row, col] + 1))//2]
+        pixel_med_diff = differences[row, col, pixel_med_index]
+        # Get the row and column of each pixel.)
 
         # In addition, decrease the index by 1 for every two diffs_to_ignore,
         # these will be saturated values in this case
-        pixel_med_diff = differences[row, col, pixel_med_index - (diffs_to_ignore / 2).astype(int)]
         # For pixels with an even number of differences the median is the mean of the two central values
         # So we need to get the
```

Comment:
Incomplete comment.    Also, does this work if there are 0 pixels with an even number of differences ?

Reply:
David,
  I reorganized the comments to better explain what is going on.
Thanks
Mike